src/_content/posts/black-python-devs.md

```diff
@@ -0,0 +1,24 @@
+---
+author: DjangoCon US Communications Team
+category: General
+published_datetime: 2024-09-13 06:00:00
+layout: post
+title: Black Python Devs Leadership Summit
+
+cover:
+    url: /assets/img/blog/bpd-summit-card-deck.jpg
+    alt: "Black Python Devs Leadership Summit"
+---
+
+Hey Friends! We know you're looking forward to travelling into Durham next week for DjangoCon US and if you're arriving on or before the 21st we have somethingi in store for you. Our Friends at [Black Python Devs](https://blackpythondevs.com) are happy to invite you to their First Ever LEADERSHIP SUMMIT and After Party presented by [Caktus Consulting Group, LLC](https://www.caktusgroup.com).
+
+We'd love for you to join us for our afternoon session when our Keynote speaker Wesley Faulkner, Interactive talks from Abigagail Afi Gbadago, Tim Schilling, and Bamhere Kudzayi, and a panel featuring members of the PSF, DSF, BPD and More.
```

Comment:
```suggestion
We'd love for you to join us for our afternoon session when our Keynote speaker Wesley Faulkner, Interactive talks from Abigagail Afi Gbadago, Tim Schilling, and Kudzayi Bamhare, and a panel featuring members of the PSF, DSF, BPD and More.
```
Everywhere I've found Kudzayi's name (our slack, the summit website, their LinkedIn, this was the was their name was spelled)